test/scenarios/update_file_via_tmp_file/scenario.js

```diff
@@ -0,0 +1,22 @@
+/* @flow */
+
+/*:: import type { Scenario } from '..' */
+
+module.exports = ({
+  side: 'local',
+  init: [{ ino: 1, path: 'file', content: 'initial content' }],
+  actions: [
+    { type: 'mv', src: 'file', dst: 'file.special-tmp' },
```

Comment:
Maybe rename file to match the specific case, .e.g. `file.ods.osl-tmp`?
And rename scenario to something like `ods_update_through_osl_tmp_file`?
Possibly explaining the case, either as a comment inside the `scenario.js`, or in a separate `README.md`?